kpi/serializers/v2/user.py

```diff
@@ -20,7 +20,7 @@ class UserSerializer(serializers.HyperlinkedModelSerializer):
     assets = PaginatedApiField(
         serializer_class=AssetUrlListSerializer
     )
-    member_since = serializers.SerializerMethodField()
+    date_joined = serializers.SerializerMethodField()
```

Comment:
I don't think we need to redeclare `date_joined` because it's already part of the model. Just adding it to `fields` should be enough. 
We can override the output though. In that case, we need to use `serializers.SerializerMethodField()`